uclouvain-centre-charles-de-visscher-pour-le-droit-international-et-europeen.csl

```diff
@@ -187,7 +187,7 @@
               <group delimiter=", ">
                 <text variable="authority"/>
                 <date form="text" variable="issued"/>
-                <text variable="title" font-style="italic"/>
+                <text variable="title" form="short" font-style="italic"/>
```

Comment:
@anfourny  note that this does exactly the same as the much more complicate choose/if/else conditional you added.

Reply:
Thanks :)